core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -2078,8 +2123,10 @@ func (o *evmTxStore) FindConfirmedTxesReceipts(ctx context.Context, finalizedBlo
 	var cancel context.CancelFunc
 	ctx, cancel = o.stopCh.Ctx(ctx)
 	defer cancel()
+
+	// note the receipts are partially loaded for performance reason
```

Comment:
I think this note may be more useful above the function in case someone doesn't go into the implementation and considers using it somewhere else. It might be helpful putting the same description/disclaimer where this is defined in the interface too.

Reply:
Thanks, good calls